docs/source/changelog.rst

```diff
@@ -5,6 +5,7 @@ Changelog
 Future Release
 ==============
     * Enhancements
+        * Change `TitleWordCount`, `PunctuationCount`, `UpperCaseCount` to use `CountString` (:pr:`183`)
```

Comment:
I'd move this to the `Changes` section. Enhancements is generally reserved for more user-related enhancements, not code improvements.